lib/workos/user_management.rb

```diff
@@ -746,6 +573,106 @@ def list_auth_factors(user_id:)
         )
       end
 
+      # Sends a verification email to the provided user.
+      #
+      # @param [String] id The unique ID of the User whose email address will be verified.
+      #
+      # @return WorkOS::UserResponse
+      sig do
+        params(
+          id: String,
+        ).returns(WorkOS::UserResponse)
+      end
+      def send_verification_email(id:)
+        response = execute_request(
+          request: post_request(
+            path: "/user_management/users/#{id}/email_verification/send",
+            auth: true,
+          ),
+        )
+
+        WorkOS::UserResponse.new(response.body)
+      end
+
+      # Verifies user email using one-time code that was sent to the user.
+      #
+      # @param [String] user_id The unique ID of the User whose email address will be verified.
+      # @param [String] code The one-time code emailed to the user.
+      #
+      # @return WorkOS::UserResponse
+      sig do
+        params(
+          user_id: String,
+          code: String,
+        ).returns(WorkOS::UserResponse)
+      end
+      def verify_email(user_id:, code:)
+        response = execute_request(
+          request: post_request(
+            path: "/user_management/users/#{user_id}/email_verification/confirm",
+            body: {
+              code: code,
+            },
+            auth: true,
+          ),
+        )
+        WorkOS::UserResponse.new(response.body)
```

Comment:
nit: empty line before returning the response

Reply:
Added on https://github.com/workos/workos-ruby/pull/241/commits/c3a7ccf1cdb934fc32ae31427dfac53aa3f087e9